content/en/docs/how-tos/interact-with-running-jobs.md

```diff
@@ -286,3 +286,243 @@ $ openshift-tests run
+  above but you can create the two directories anywhere where you have access.
+
+  ```bash
+  mydir=`pwd`/tmp
+  mkdir -p $mydir/logs/artifacts/junit
+  ```
+
+* For aws, do these:
+
+  * determine the region and zone of your cluster so you can use those values in your `--provider` parameter on the `openshift-tests` command:
+
+    ```bash
+    $ cat j.json |jq '.items[]|.metadata.labels'|grep topology.kubernetes.io/region|cut -d : -f 2| sort -u
+    "us-west-1",
+
+    $ cat j.json |jq '.items[]|.metadata.labels'|grep topology.kubernetes.io/zone|cut -d : -f 2| sort -u
+    "us-west-1a"
+    "us-west-1b"
+    ```
+
+    For the example above, you will use this:
+
+    ```bash
+    --provider '{"type":"aws", "region":"us-west-1", "zone":"us-west-1","multizone":true,"multimaster":true}'
+    ```
+
+
+* For gcp, do these:
+
+  * get a gcp credentials file and setup the environment variable so openshift-tests can find it:
+
+    ```bash
+    export GOOGLE_APPLICATION_CREDENTIALS=<your gce credentials>
+    ```
+  * determine the region of your cluster so you can use it in your `--provider` paramater on the `openshift-tests` command:
+
+    ```bash
+    $ cat j.json |jq '.items[]|.metadata.labels'|grep topology.kubernetes.io/region|cut -d : -f 2| sort -u
+    "us-west-1",
+    ```
+
+    For the example above, you will use this:
+
+    ```bash
+    --provider '{"type":"gce", "region":"us-west-1","multizone": true,"multimaster":true,"projectid":"<yourProjectID>"}'
+    ```
+
+### Running `openshift-tests`
+
+Once the above requirements are met, run the `openshift-tests` binary for one of these scenarios:
+
+* Running the "openshift/conformance/serial" conformance suite
+
+  Using an aws cluster, here's an example command to run the openshift/conformance/serial tests:
+
+  ```bash
+  ~/openshift-tests run openshift/conformance/serial \
+         --provider '{"type":"aws", "region":"us-west-1", "zone":"us-west-1","multizone":true,"multimaster":true}' \
+         -o $mydir/logs/artifacts/e2e.log --junit-dir $mydir/logs/artifacts/junit
+  ```
+
+* Run the upgrade test:
+
+  Using an aws cluster, here's an example command to run an upgrade tests:
+
+  ```bash
+  ~/openshift-tests run-upgrade all --to-image 4.11.0-0.nightly-2022-06-15-161625 \
+         --options '' \
+         --provider '{"type":"aws", "region":"us-west-1", "zone":"us-west-1","multizone":true,"multimaster":true}' \
+         -o $mydir/logs/artifacts/e2e.log --junit-dir $mydir/logs/artifacts/junit
+  ```
+
+  Using an gcp cluster, here's an example command to run an upgrade tests:
+
+  ```bash
+  ~/openshift-tests run-upgrade all --to-image 4.11.0-0.nightly-2022-06-15-161625 \
+         --options '' \
+         --provider '{"type":"gce", "region":"us-west-1","multizone": true,"multimaster":true,"projectid":"<yourProjectID>"}' \
+         -o $mydir/logs/artifacts/e2e.log --junit-dir $mydir/logs/artifacts/junit
+  ```
```

Comment:
Is there a base non-serial non-upgrade e2e conformance test suite? That would be great to add here, it might be very common for people looking to do this during development.

Reply:
Ah, good idea.  Let me try to find one ...